client/src/test/scala/org/http4s/client/JettyScaffold.scala

```diff
@@ -6,40 +6,31 @@ import javax.servlet.http.HttpServlet
 import org.eclipse.jetty.server.{Server => JServer, ServerConnector}
 import org.eclipse.jetty.servlet.{ServletHolder, ServletContextHandler}
 import org.http4s.Http4sSpec
+import org.specs2.mutable.SpecificationLike
+import org.specs2.specification.BeforeAfterAll
 import org.specs2.specification.core.Fragments
 
 import scala.concurrent.duration._
 
-
-abstract class JettyScaffold(name: String) extends Http4sSpec {
+trait JettyScaffold extends SpecificationLike {
 
   private val server = new JServer()
+  var address: InetSocketAddress = null
```

Comment:
Oh, I saw plenty of NPEs last night.  We have to be careful to make the fragments lazy.  I think this is why the code previously initialized the server closer to the fragments, but it's also why we started a bazillion.